libobs/cmake/libobsConfig.cmake.in

```diff
@@ -0,0 +1,19 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/@TARGETS_EXPORT_NAME@.cmake")
+check_required_components("@PROJECT_NAME@")
+
+if(MSVC)
+  find_package(w32-pthreads REQUIRED)
+endif()
+
+find_package(Threads REQUIRED)
+
+set(LIBOBS_PLUGIN_DESTINATION "@PACKAGE_OBS_PLUGIN_DESTINATION@")
+set(LIBOBS_PLUGIN_DATA_DESTINATION "@PACKAGE_OBS_DATA_DESTINATION@/obs-plugins")
+
+# set_and_check(LIBOBS_PLUGIN_DESTINATION "${_DIR}/@OBS_PLUGIN_DESTINATION@")
+# set_and_check(LIBOBS_PLUGIN_DATA_DESTINATION
+# "${_DIR}/@OBS_DATA_DESTINATION@/obs-plugins")
```

Comment:
If these aren't needed, they can be removed.

Reply:
Removed.